### ISSUE TITLE
Cobblemon integration

### ISSUE THREAD

**Issue by baebypowder** [Pandarix/BetterArcheology#101]
Would it be possible to add support for Cobblemon's archeology features? Or possibly allowing the loot tables for BetterArcheology to be edited via its config file

**Comment by Fayvira-DraGon** [comment]
Yes, please! This would be amazing! @Pandarix do you have a discord? If you wanted to crowd-source simple fossil statues of the pokémon, I am sure people would be stoked especially from the official Cobblemon discord server (cobblemon.com)!
